components/migrate/steps/SignLoader1.vue

```diff
@@ -0,0 +1,217 @@
+        </p>
+      </div>
+    </div>
+    <div class="is-flex is-size-7">
+      <div class="v-border"></div>
+      <div class="mb-4">
+        <p v-if="step1Iterations">{{ step1Iterations }}/2 Left</p>
+        <p v-else>Done</p>
+      </div>
+    </div>
+    <div class="is-flex is-size-7">
+      <div class="v-border"></div>
+      <div class="mb-4 is-flex">
+        <div class="mr-4">
+          <NeoIcon v-if="step1Iterations === 0" v-bind="iconSuccess" />
+          <NeoIcon v-else-if="step1Iterations === 1" v-bind="iconLoading" />
+          <NeoIcon v-else v-bind="iconIdle" />
+        </div>
+        <div>
+          <p>Create Collection</p>
+          <nuxt-link
+            v-if="step1Iterations === 0"
+            target="_blank"
+            class="has-text-k-blue"
+            :to="`/${client}/collection/${nextId}`">
+            View Tx <NeoIcon icon="arrow-up-right" />
+          </nuxt-link>
+        </div>
+      </div>
+    </div>
+    <div class="is-flex is-size-7">
+      <div class="v-border"></div>
+      <div class="mb-4 is-flex">
+        <div class="mr-4">
+          <NeoIcon v-if="step1Iterations === 0" v-bind="iconSuccess" />
+          <NeoIcon v-else v-bind="iconIdle" />
+        </div>
+        <div>
+          <p>Automated Asset Preparation</p>
+          <p>No Signature Needed</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import type { Prefix } from '@kodadot1/static'
+import { NeoIcon } from '@kodadot1/brick'
+import {
+  type Steps,
+  iconIdle,
+  iconLoading,
+  iconSuccess,
+  useCollectionReady,
+} from '@/components/migrate/utils'
+import { useStatemineNewCollectionId } from '@/composables/transaction/mintCollection/useNewCollectionId'
+import { createArgsForNftPallet } from '@/composables/transaction/mintCollection/utils'
+import waifuApi from '@/services/waifu'
+
+const { accountId } = useAuth()
+const { client } = usePrefix()
+const { apiInstance } = useApi()
+const { nextCollectionId } = useStatemineNewCollectionId()
+const { howAboutToExecute, status } = useMetaTransaction()
+const { $consola } = useNuxtApp()
+const route = useRoute()
+const router = useRouter()
+
+const to = route.query.destination as Prefix
+const from = route.query.source as Prefix
+const fromAccountId = route.query.accountId?.toString()
+
+const api = await apiInstance.value
+const { steps, updateSteps } = inject('steps') as {
+  steps: Ref<Steps>
+  updateSteps: (step: Steps) => void
+}
+
+const nextId = ref('0')
+const relocationsBody = ref({})
+
+const ITERATIONS = 2
+const step1Iterations = ref(ITERATIONS)
+
+const { collections } = await useCollectionReady(from, fromAccountId)
+const fromCollection = collections.value.find(
+  (collection) => collection.id === route.query.collectionId,
+)
+
+const deleteRelocations = async () => {
+  const relocationsId = `${from}-${fromAccountId}`
+  return await waifuApi(`/relocations/${relocationsId}`, {
```

Comment:
i'm confused why waifu api?

Reply:
If you are asking about why `import waifuApi from '@/services/waifu'` is because there is already an `api` variable in this file

If you ask why `/relocations` is on `waifu` workers, I guess @vikiival can answer this. Viik is helping me a lot with the backend and the script in this PR